src/components/DappBrowser/BrowserTab.tsx

```diff
@@ -616,13 +617,13 @@ export const BrowserTab = React.memo(function BrowserTab({ tabId, tabIndex, inje
 
   const swipeToCloseTabGestureHandler = useAnimatedGestureHandler<PanGestureHandlerGestureEvent>({
     onStart: (_, ctx: { startX?: number }) => {
-      if (!tabViewVisible?.value) return;
```

Comment:
this should be removed once we fix the disabling of the gesture handlers outside of the tab view

probably currently allows the tab to get stuck if while dragging a tab you exit the tab view

we could even remove the early return in `onEnd` now and add the `!tabViewVisible?.value` condition where I added `!isEmptyTab`:

```
onEnd: (e, ctx: { startX?: number }) => {
      const xDelta = e.absoluteX - (ctx.startX || 0);
      setNativeProps(scrollViewRef, { scrollEnabled: !!tabViewVisible?.value });

      const isBeyondDismissThreshold = (xDelta < -(TAB_VIEW_COLUMN_WIDTH / 2 + 20) && e.velocityX <= 0);
      const isFastLeftwardSwipe = e.velocityX < -500;
      const shouldDismiss =  !tabViewVisible?.value && !isEmptyState && (isBeyondDismissThreshold || isFastLeftwardSwipe);

      if (shouldDismiss) {
      
      // rest the same
```